cns/middlewares/k8sSwiftV2_linux_test.go

```diff
@@ -342,10 +343,10 @@ func TestSetRoutesSuccess(t *testing.T) {
 		} else {
 			assert.Equal(t, ipInfo.SkipDefaultRoutes, false)
 		}
-
 	}
+
 	for i := range podIPInfo {
-		assert.DeepEqual(t, podIPInfo[i].Routes, desiredPodIPInfo[i].Routes)
+		reflect.DeepEqual(podIPInfo[i].Routes, desiredPodIPInfo[i].Routes)
```

Comment:
Preference towards `github.com/google/go-cmp/cmp.Equal` over `reflect.DeepEqual` (it's a drop-in replacement)